test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -52,7 +52,7 @@ file.on('close', function() {
   //assert.equal(fs.readFileSync(fn), fileContent);
 });
 
-var file3 = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
+var file3 = fs.createReadStream(fn, {encoding: 'utf8'});
```

Comment:
That's going to be a problem, I think. We have no way of knowing how many users out there may be using the `Object.create({})` approach.